source/jormungandr/jormungandr/scenarios/experimental.py

```diff
@@ -148,8 +156,8 @@ def call_kraken(self, request_type, request, instance, krakens_call):
             self.nb_kraken_calls += 1
             direct_path = self._get_direct_path(instance,
                                                 dep_mode,
-                                                request['origin'],
-                                                request['destination'],
+                                                g.requested_origin,
```

Comment:
Direct path takes a place instead of an object because valhalla need coords, and there is no coords in a stop area id.